test-suite/src/test/scala/org/scalajs/testsuite/javalib/MathTest.scala

```diff
@@ -15,6 +15,51 @@ object MathTest extends JasmineTest {
 
   describe("java.lang.Math") {
 
+    it("should respond to `abs`") {
+      expect(Math.abs(0)).toBe(0)
+      expect(Math.abs(42)).toBe(42)
+      expect(Math.abs(-42)).toBe(42)
+      expect(Math.abs(0.0)).toBe(0.0)
+      expect(Math.abs(-0.0)).toBe(0.0)
```

Comment:
You have to use `expect(Math.abs(-0.0).equals(0.0)).toBeTruthy` here. Same for the line about about `Math.abs(0.0)`